### DISCUSSION TITLE
关于部分E.G.O名称的考据，和修改建议

### DISCUSSION THREAD

**Discussion by Iansang10032** [LocalizeLimbusCompany/LocalizeLimbusCompany#54] [想法 | Ideas]
1、关于李箱、良秀、罗佳的第二E.G.O装备，韩文原作名称为4번째 성냥불（第四根火柴）取自《安徒生童话——卖火柴的小女孩》小女孩燃尽的最后一根火柴而获得永恒的光明和失去生命的典故，故《脑叶公司》的译名终末火柴之光要比《废墟图书馆》的译名终末之光更为恰当，也更为贴合原作翻译。终末之光这个译名实在有些舍本逐末，烦请制作组考虑一下。
2、关于罗佳第三E.G.O装备，韩文原作名称为얼음다리（冰桥，冰腿，冰爪）取自俄罗斯民间传说芭芭雅嘎婆婆，
雾凇之爪虽然很有意境，但结合《脑叶公司》原作，大部分的E.G.O的译名都是简化词，目灯，拟态，圣宣，且大部分武器没有形态描述词，
并非拟态大刀，正裁之剑，当然有盈泪之剑这样的特例。不过我还是坚持简化翻译比较符合前作翻译特点，故坚持认为冰爪也许更为合适。
3、关于浮士德第二E.G.O诅咒钉的来源异想体，很明显与《脑叶公司》中的快乐泰迪是一对对应异想体，所以是不是翻译取快乐的反义词痛苦比痛痛更加合适一点，也算与前作呼应。希望制作组可以考虑一下

**Comment by N071** [comment]
第一点：
- 图书馆翻译为“终末之光”是由于对四个字的执念
- 零协会内部讨论在之前已认同是“终末火柴之光”
- Github处当时未跟进
- 目前已按照斩打突的最终意见进行修改，改为“终末火柴之光”

后续两点待讨论，请稍等待结果

**Comment by N071** [comment]
# 雾凇之爪

说在前面...

* 我们全面参考了韩文原文与英日译文。
* 我们知晓该名称典故来自于俄罗斯民间童话。

关于简化词：

* LCB中大部分译名均继承自LOR，仅有少部分继承自LC
* 因此，我们决定(在判断基础上)以LOR的翻译风格以及字句构成为主。这造就了我们现在**以四字长度为标准**，在格式公整，语义完全的情况下再采用其他长度的翻译体系。
* 本作中，比起一个人身上可以装备的两件装备，EGO更像是一个较大的技能。因此，我们认为EGO在**某种程度上**应该参考技能的翻译标准。

关于“雾凇之爪”本身与你的提议：

* “雾凇”来自英文原文Rime。我们在参考其他原文后决定**以艺术修饰为主要目的**，**不破坏原意为底线的**，沿用英文的这一扩展意。
* 正如以上所说，我们不会受LC的翻译格式影响，也不会受LOR翻译格式**太多**影响。参考到LCB本作的多样性，我们的翻译格式是相对具体问题具体分析的。因此，我们不认同“参考LC原作翻译”是一个充分的理由。
* 是否选择“参考LC原作翻译”涉及到我们整体的翻译哲学。如若你希望推进这一标准，请你审慎的对目前的所有翻译进行LC化的处理后一并提出供各位审阅。目前，“雾凇之爪”服从零协会目前现行的翻译体系。
* 很多名称无法翻译成“参考LC原作翻译”的简化词。Rime Shank可能不属于其中之一，但仍有大量翻译需要四字(甚至更多)才能体现其完整的意义。
# 痛痛泰迪

关于异想体...

* 实际上，在我们刚刚接触拟态的翻译时，有些人不认为那些扭曲的判官与拟态有关(我们不在此评价他们对剧情以及细节的理解程度)。当时，最有力的证据是异想体编号，说服了任何不同的声音。
* 同样的，也有人一直认为粉舞鞋是红舞鞋的某一个亚种或衍生异想体，但其编号从根本上完全不同，也使这些人看清了本质。
* 因此，解释两个异想体之间是否存在关联的最优方式是提供明确的编号考据，而非“很明显”三字中承载的潜在猜测性质。
* 另外，我们没法获取该EGO来源异想体“痛痛泰迪”的具体编号。如果能够找到新的考据，我们很乐意服从。
* _痛痛泰迪真的很可爱。_

**Reply by Iansang10032** [reply]
事实上亚种这个设定被取消确实也是我一直强调的并认同的观点，但是根据黑檀皇后的苹果和白雪公主的苹果这两个异想体的关系来看，
异想体对应关系还是存在的，所以黑檀皇后的翻译要比黑檀女王的翻译更为准确。
好吧确实痛痛泰迪的异想体还没有出现观察报告，如果有类似黑檀皇后的苹果观察报告中的对应关系描述的话，
还麻烦组织重新考虑痛痛泰迪的翻译。

**Comment by Iansang10032** [comment]
关于雾凇:
第一为什么是参考英文而不是参考韩文呢，我们是有比较靠谱的韩翻大佬的，
第二雾凇是树枝遇到气温骤降而产生结冰的现象，事实上雾凇的核心是树枝，但是冰爪这个完全与树枝不搭边的不是吗？
       即使是翻译成冰之爪或者寒冰之爪，都是可以令人接受的。还望组织再次斟酌一下。

**Comment by N071** [comment]
经过零协会讨论
雾凇之爪已确定为冰结之爪

另外，不论是出自零协会的译名或者是零协会本身的处事态度，都**没有**什么是**不可以接受**的。我们希望各位抱着尊重以及征询的心态进行译名讨论，审慎地选取与他人讨论的用词。
